index.js

```diff
@@ -107,6 +108,13 @@ function supportsHyperlink(stream) {
 		return version.major > 0 || version.minor >= 50;
 	}
 
+	switch (TERM) {
+		case 'alacritty':
```

Comment:
Can't you check the `$TERM_PROGRAM_VERSION` too?

Reply:
Looks like `TERM_PROGRAM_VERSION` is not set by  `alacritty`.  https://github.com/alacritty/alacritty/issues/781

-------
confirm the term window I'm in is alacritty
```
xprop -id $(xdotool getwindowfocus)|grep ^WM_CLASS
  WM_CLASS(STRING) = "Alacritty", "Alacritty"
```

only TERM and COLORTERM in the environment
```
env |grep TERM
  COLORTERM=truecolor
  TERM=alacritty
```

```
alacritty --version
  alacritty 0.15.0 (53395536)
```